app/models/miq_ae_yaml_export.rb

```diff
@@ -201,6 +201,8 @@ def get_method_filename(method_obj)
       "#{method_obj.name}#{FILE_EXTENSIONS[method_obj.language]}"
     elsif method_obj['location'] == 'uri'
       "#{method_obj.name}.uri"
+    elsif method_obj['location'] == 'expression'
+      "#{method_obj.name}.expr"
```

Comment:
Is the expression data being saved in `YAML` format?  If so do we need a new file type or can we just use `.yaml`?